### DISCUSSION TITLE
2006/08/22/%E6%95%B4%E5%90%88-wiki-blog-%E5%92%8C-forum

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#110] [Announcements]
# 2006/08/22/%E6%95%B4%E5%90%88-wiki-blog-%E5%92%8C-forum

现在，wiki、blog 和论坛都已经成为我工作中不可缺少的工具了。他们三个各有优缺点：

https://blog.qizhen.xyz/2006/08/22/%E6%95%B4%E5%90%88-wiki-blog-%E5%92%8C-forum

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Jian
2008年12月10日 19:51 编辑
你好!我非常感兴趣你的关于整合wiki, blog 和 forum的想法.你那里是否还有近一步详细的方案介绍,希望你能我联系.我的电邮:zhangjj_de@hotmail.com. 谢谢!

回复
习习
2007年02月8日 15:38 编辑
路过
系统只不过是工具,内容才是王道

回复
rongqiang
2006年10月5日 14:05 编辑
从高工在EDN的博客上看到了这位高手，碰巧又有我感兴趣的话题，所以留下一个脚印。据我所知。wiki是支持评论的。但是wiki的评论礼仪和forum略有不同。我也看了您写的为公司搭建文件系统的文章。其实我一直想要在EDN网站上作这个尝试。当然，我自己的一个朋友圈子，也想要做成wiki站点，分享大家的一些心得和经验。:D

回复
永青
2006年08月28日 14:46 编辑
嗯，这是个好主意，说不定可以做一做，我想技术上并不太难

回复
